packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -507,5 +511,45 @@ angular.module('managerApp').controller(
           this.$scope.loaders.incident = false;
         });
     }
+
+    /**
+     * Retrieve modem detail for exchange if available
+     */
+    getModemExchange() {
+      return this.OvhApiXdsl.Modem()
+        .v6()
+        .getComfortExchange({
+          xdslId: this.$stateParams.serviceName,
+        })
+        .$promise.then((result) => {
+          this.$scope.access.xdsl.canExchange = true;
+          this.$scope.access.xdsl.newModel = result.newModel
+            .replace(/\./g, ' ')
+            .toUpperCase();
+          this.$scope.access.xdsl.priceHT = result.price.value;
+          this.$scope.access.xdsl.priceTTC =
+            result.price.value + (result.price.value * 20) / 100;
```

Comment:
IMO, we souldn't calculate price, it should come from the API.

Reply:
the API just send the HT price, not the TTC price

Reply:
In addition to @cbourgois comment, here we assume that we are in FR subsidiary, but what if we decide that this product is available elsewhere ? The tax calculation would probably be different

Reply:
I guess it all sums up to: can this be handled by `/order` instead ?

Reply:
I asked to update the API to retrieve the TTC price.